src/core/layout/qgslayoutexporter.cpp

```diff
@@ -1466,6 +1436,169 @@ bool QgsLayoutExporter::georeferenceOutputPrivate( const QString &file, QgsLayou
   return true;
 }
 
+QString nameForLayerWithItems( const QList< QGraphicsItem * > items, unsigned int layerId )
```

Comment:
Any reason why this function is not part of the QgsLayoutExporter class?

Reply:
It's just a little implementation detail -- I don't see any need to expose it to the class.